middlewares/logger-middleware.ts

```diff
@@ -0,0 +1,17 @@
+import {PolarisMiddleware} from "./polaris-middleware";
+import {inject, injectable} from "inversify";
+import {PolarisLogger} from "@enigmatis/polaris-logs/dist/src/polaris-logger";
+import {InjectableLogger} from "..";
+
+@injectable()
+export class LoggerMiddleware implements PolarisMiddleware {
+    @inject("InjectableLogger") polarisLogger: InjectableLogger;
+
+    preResolve(resolve, root, args, context, info) {
```

Comment:
check what those parameters contain, so we can log more usful information, maybe on the content of the context